db/migrate/20240112021335_set_max_clean_orphan_uploads_grace_period_hours.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SetMaxCleanOrphanUploadsGracePeriodHours < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec <<~SQL
+        UPDATE site_settings SET value = '168' WHERE name ='clean_orphan_uploads_grace_period_hours' and value > '168';
```

Comment:
For `value > '168'` I think you may have to cast the value as int first? Otherwise it would be just comparing on string length...I think

Reply:
Hmm.. the migration was successful. 🧐

Reply:
I think without the cast it is still OK.

```
[1] pry(main)> "168">"169"
=> false
[2] pry(main)> "168">"168"
=> false
[3] pry(main)> "168">"167"
=> true
```

But I will cast to make it clearer 👍 ✨